### DISCUSSION TITLE
Account Banned

### DISCUSSION THREAD

**Discussion by sureshraj12** [community/community#64770] [GitHub Education]
### Hi everyone,

Hi Team 
I created my account 1 week ago my account is banned today kindly tell me how should i make it unban it is very important for me to get that account i have many personal projects there![image](https://github.com/community/community/assets/137879832/fea40e8e-e6a9-42d9-ab11-4df9da87583c)

**Comment by arafajs** [comment]
Contact support is the best thing to do here.
